src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -820,6 +820,7 @@ TEST_CASE("request_rate_sequence")
 TEST_CASE("request rate sequence: verify inferences across sequences")
 {
   PerfAnalyzerParameters params;
+  params.DEB_new_option = true;
```

Comment:
Update test name or comments to imply that this is only testing behavior of the new mode

Reply:
done